posts/tradeoff-of-defining-types-as-subobjects.md

```diff
@@ -0,0 +1,119 @@
+---
+author: 'Yaël Dillies'
+category: 'Papers'
+date: 2024-07-25 14:00:00 UTC+03:00
+description: ''
+has_math: true
+link: ''
+slug: tradeoff-of-defining-types-as-subobjects
+tags: ''
+title: Tradeoffs of defining types as subobjects
+type: text
+---
+
+It often happens in formalisation that a type of interest is a subobject of another type of interest. For example, the unit circle in the complex plane is naturally a submonoid (nevermind that it is actually a subgroup, we currently can't talk about subgroups of fields).
```

Comment:
I don't know how to make footnotes, but I mean the sentence in bracket to be a footnote

Reply:
According to [this page](https://github.blog/changelog/2021-09-30-footnotes-now-supported-in-markdown-fields/) footnotes can be done in markdown as
```
Here is a simple footnote[^1]. With some additional text after it.

[^1]: My reference.
```

I'm not sure that I ever tried it, though.

Reply:
Seems to work in preview[^1]

[^1]: at least